retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
> It's not. The explicitly empty label is gone after ingestion, you federate, new instance attached. This adds up to 3 already.

That's the general federation case, it's the same as for the PGW and I think they'll all end up being handled the same way.

Everything thus far is us needing a way to signal to the server not to attach an `instance` label to certain metrics.